src/style.css

```
@@ -0,0 +1,74 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-flow: column-reverse wrap-reverse;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px ;
+  color: #fff;
+}
+
+.block {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex: 1 0 300px ;
```

add flex-basis in percent
```suggestion
  flex: 1 0 100% ;
```